src/main/kotlin/com/makarytskyi/rentcar/dto/order/OrderResponse.kt

```diff
@@ -15,20 +15,20 @@ data class OrderResponse(
 ) {
 
     companion object {
-        fun from(order: Order, carPrice: Int?): OrderResponse {
+        fun from(mongoOrder: MongoOrder, carPrice: Int?): OrderResponse {
             val daysBetween = ChronoUnit.DAYS.between(
-                order.from?.toInstant()?.atZone(ZoneId.systemDefault())?.toLocalDate(),
-                order.to?.toInstant()?.atZone(ZoneId.systemDefault())?.toLocalDate()
+                mongoOrder.from?.toInstant()?.atZone(ZoneId.systemDefault())?.toLocalDate(),
+                mongoOrder.to?.toInstant()?.atZone(ZoneId.systemDefault())?.toLocalDate()
```

Comment:
It seems such logic for **daysBetween** variable is the same, as in **AggregatedOrderResponse** class